src/Microsoft.Health.Fhir.Core/Features/Search/Expressions/Expression.cs

```diff
@@ -94,7 +94,7 @@ public static ChainedExpression Chained(string[] resourceTypes, SearchParameterI
         /// <returns>A <see cref="IncludeExpression"/> that represents an include on <param name="targetResourceType"> through <paramref name="referenceSearchParameter"/>.</param></returns>
         public static IncludeExpression Include(string[] resourceTypes, SearchParameterInfo referenceSearchParameter, string sourceResourceType, string targetResourceType, IEnumerable<string> referencedTypes, bool wildCard, bool reversed, bool iterate)
         {
-            return new IncludeExpression(resourceTypes, referenceSearchParameter, sourceResourceType, targetResourceType, referencedTypes, wildCard, reversed, iterate);
+            return new IncludeExpression(resourceTypes, referenceSearchParameter, sourceResourceType, targetResourceType, referencedTypes, wildCard, reversed, iterate, null);
```

Comment:
I think the PR looks great.
My only suggestion is using a named argument when passing the value _null_ at this line. 
Just because, if tomorrow we add a new nullable argument to IncludeExpression, the compiler will crash saying that it does not know what it's the correct argument corresponding to this null value.

Reply:
You can ignore this comment as well and merge the changes.